src/isar_exr/robotinterface.py

```diff
@@ -141,8 +144,13 @@ def mission_status(self) -> MissionStatus:
         except NoMissionRunningException:
             # This is a temporary solution until we have mission status by mission id
             return MissionStatus.Successful
-        except Exception as e:
-            raise RobotException from e
+        except Exception:
+            message: str = "Could not get status of running mission"
+            self.logger.error(message)
+            raise RobotException(
```

Comment:
Why not raise `RobotMissionStatusException` and just pass the error description?